sympy/series/limits.py

```diff
@@ -168,6 +168,16 @@ def doit(self, **hints):
         if e.is_Order:
             return Order(limit(e.expr, z, z0), *e.args[1:])
 
+        #Check to see if e is a simple power series, like (-1/2)**x
+        if e.is_Pow and len(e.free_symbols) == 1:
+            k = e.free_symbols.pop()
+            if k == z:
```

Comment:
Yeah, that was poorly thought out--I'll rewrite to only accept positive multiples of `z`.